res/values/strings.xml

```diff
@@ -258,6 +258,7 @@
     <string name="login_title">Login</string>
     <string name="login_header">Login to your server</string>
     <string name="login_subheader">For known email providers additional settings are setup automatically. Sometimes IMAP needs to be enabled in the web frontend. Consult your email provider or friends for help.</string>
+    <string name="login_no_servers_hint">Delta Chat has no servers, your data stays on your device!</string>
```

Comment:
on delta.chat the first half sentences was marked as a germanism and fixed to read: Delta Chat doesn’t have their own servers. 

It's a bit long though and i wonder if we could say "There are no Delta Chat servers." both here and on the website ;) 
 
FWIW I also a user on twitter use "delta chat requires no sign up" -- thought it makes sense because "sign up" is what is in user's faces.  Then again, someone could claim "no sign up" and still create some cloud account using the same credentials as your e-mail account.

Reply:
so bad that "Delta Chat has no servers" is no english :)

but i would also go for "There are no Delta Chat servers." then. doing a pr